File: Selenium/Project/ProjectActivity3.py
# Import webdriver from selenium

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Set up the Firefox Driver with WebDriverManger
with webdriver.Firefox(service=service) as driver:
    # Navigate to the URL

    driver.get("https://alchemy.hguy.co/jobs/")
    headerImage = driver.find_element(By.XPATH,"//img[@class='attachment-large size-large wp-post-image']")
    imageUrl = headerImage.get_attribute("src")
    print("header image url: " + imageUrl)

File: Selenium/Project/ProjectActivity3.java
package ProjectActivities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class ProjectActivity3 {
    WebDriver driver;
    @BeforeClass
    public void setUp() {
        WebDriverManager.firefoxdriver().setup();
        // Create a new instance of the Firefox driver
        driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs/");
    }

     @Test
     public void url_header(){
        WebElement headerImage  = driver.findElement(By.xpath("//img[@class='attachment-large size-large wp-post-image']"));
        String imageUrl = headerImage.getAttribute("src");
        System.out.println("header image url: " +imageUrl);

    }

    @AfterClass
    public void tearDown(){
        driver.quit();
    }
}